.github/workflows/experimental.yaml

```diff
@@ -0,0 +1,119 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: ⚗️ experimental
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+  schedule:
+    - cron: 0 0 * * 0
+
+env:
+  GO_VERSION: ^1.22.0
+
+jobs:
+  affected:
+    name: Finding affected tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    outputs:
+      nodejs: ${{ steps.nodejs.outputs.packages }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - run: go install ./cmd/affected
```

Comment:
nit/thought: I'm not sure whether `go install` or `go build` is a better option for this system. If the binary is going to be reused several times per session, then `build` might be the better option (since all extraneous compiled code gets thrown away).

Reply:
I just changed it to use build instead of install.